src/main/scanLogic/scanRunners/jasRunner.ts

```diff
@@ -127,66 +75,26 @@ export abstract class JasRunner {
-        if (!this.validateSupported()) {
-            this._logManager.logMessage(this._scanType + ' runner could not find binary to run', 'WARN');
-            return false;
-        }
         if (this._config.ShouldSkipScanner(this._scanType)) {
             this._logManager.debug('Skipping ' + this._scanType + ' scanning');
             return false;
         }
         return true;
     }
 
-    /**
-     * Validates that the binary exists and can run
-     * @returns true if supported false otherwise
-     */
-    public validateSupported(): boolean {
-        return this._binary.isExists();
-    }
-
     /**
      * Execute the cmd command to run the binary with given arguments and an option to abort the operation.
      * @param checkCancel           - Check if should cancel
      * @param args                  - Arguments for the command
      * @param executionLogDirectory - Directory to save the execution log in
      */
     protected async executeBinary(checkCancel: () => void, args: string[], executionLogDirectory?: string): Promise<void> {
-        await RunUtils.runWithTimeout(JasRunner.TIMEOUT_MILLISECS, checkCancel, {
-            title: this._binary.name,
-            task: this.executeBinaryTask(args, executionLogDirectory)
-        });
-    }
-
-    /**
-     * Execute the cmd command to run the binary with given arguments
-     * @param args  - the arguments for the command
-     * @param executionLogDirectory - the directory to save the execution log in
-     */
-    private async executeBinaryTask(args: string[], executionLogDirectory?: string): Promise<any> {
-        let command: string = '"' + this._binary.fullPath + '" ' + args.join(' ');
-        this._logManager.debug("Executing '" + command + "' in directory '" + this._runDirectory + "'");
-        let std: any = await ScanUtils.executeCmdAsync(command, this._runDirectory, this.createEnvForRun(executionLogDirectory));
-        if (std.stdout && std.stdout.length > 0) {
-            this.logTaskResult(std.stdout, false);
-        }
-        if (std.stderr && std.stderr.length > 0) {
-            this.logTaskResult(std.stderr, true);
-        }
-    }
-
-    private logTaskResult(stdChannel: string, isErr: boolean) {
-        let text: string = "Done executing '" + Translators.toAnalyzerTypeString(this._scanType) + "' with log, log:\n" + stdChannel;
-        if (this._verbose) {
-            console.log(text);
-        }
-        this._logManager.logMessage(text, isErr ? 'ERR' : 'DEBUG');
+        await this._analyzerManager.runWithTimeout(checkCancel, args, executionLogDirectory);
     }
 
     protected logStartScanning(request: AnalyzeScanRequest): void {
         this._logManager.logMessage(
-            `Scanning directories ' ${request.roots} + ', for ${this._scanType} issues. Skipping folders: ${request.skipped_folders}`,
+            `Scanning directories '${request.roots}', for ${this._scanType} issues. Skipping folders: ${request.skipped_folders}`,
```

Comment:
In sast the "skipped_folder" is "exclude_patterns". It looks bad in the log:
```
[DEBUG - 5:17:01 PM] Scanning directories '/Users/yahavi/code/bad-guys', for sast issues. Skipping folders: undefined
[DEBUG - 5:17:01 PM] Input YAML:
scans:
  - type: sast
    roots:
      - /Users/yahavi/code/bad-guys
    exclude_patterns:
      - '**/*.venv*/**'
```

Reply:
It is already been fixed at https://github.com/jfrog/jfrog-vscode-extension/pull/433/